rootfs/bin/create-bucket

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+import os
+
+import boto
+import json
+import swiftclient
+from boto import config as botoconfig
+from boto.s3.connection import S3Connection, OrdinaryCallingFormat
+from oauth2client.service_account import ServiceAccountCredentials
+from gcloud.storage.client import Client
+from gcloud import exceptions
+from azure.storage.blob import BlobService
+
+def bucket_exists(conn, name):
+    bucket = conn.lookup(name)
+    if not bucket:
+        return False
+    return True
+
+bucket_name = os.getenv('BUCKET_NAME')
+
+if os.getenv('REGISTRY_STORAGE') == "s3" and os.getenv('REGISTRY_STORAGE_S3_BACKEND') != 'minio':
+    conn = boto.connect_s3()
+
+    if not bucket_exists(conn, bucket_name):
+        conn.create_bucket(bucket_name)
+
+elif os.getenv('REGISTRY_STORAGE') == "gcs":
+    scopes = ['https://www.googleapis.com/auth/devstorage.full_control']
+    credentials = ServiceAccountCredentials.from_json_keyfile_name(os.getenv('REGISTRY_STORAGE_GCS_KEYFILE'), scopes=scopes)
+    with open(os.getenv('REGISTRY_STORAGE_GCS_KEYFILE')) as data_file:
+        data = json.load(data_file)
+    conn = Client(credentials=credentials, project=data['project_id'])
+    exists = True
+    try:
+        conn.get_bucket(bucket_name)
+    except exceptions.NotFound:
+        exists = False
+    except:
+        raise
+    if not exists:
```

Comment:
You could do away with the temp `exists` var and use [`else:`](https://docs.python.org/3/tutorial/errors.html#handling-exceptions) here instead.